apollo-router/src/plugins/include_subgraph_errors/config.rs

```diff
@@ -0,0 +1,227 @@
+use std::collections::HashMap;
+use std::fmt;
+
+use schemars::JsonSchema;
+use serde::Deserialize;
+use serde::Serialize;
+use serde::de::Deserializer;
+use serde::de::MapAccess;
+use serde::de::Visitor;
+use serde::de::{self};
+
+/// Configuration for exposing errors that originate from subgraphs
+#[derive(Clone, Debug, JsonSchema, Default, Deserialize)]
+#[serde(rename_all = "snake_case", deny_unknown_fields, default)]
+pub(crate) struct Config {
+    /// Configuration for all subgraphs, and handles subgraph errors
```

Comment:
This sentence reads weird with the `, and handles subgraph errors` at the end. Could you make this more explicit?

Reply:
[e549e03](https://github.com/apollographql/router/pull/7164/commits/e549e038c362c07786f35c6739b7c96446571e1b)